docs/tutorials/java/mxnet_java_on_intellij.md

```diff
@@ -0,0 +1,210 @@
+# Run MXNet Java Examples Using the IntelliJ IDE (macOS)
+
+This tutorial guides you through setting up a simple Java project in IntelliJ IDE on macOS and demonstrates usage of the MXNet Java APIs. 
+
+## Prerequisites:
+To use this tutorial you need the following pre-requisites:
+
+- [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+- [Maven](https://maven.apache.org/install.html)
+- [OpenCV](https://opencv.org/)
```

Comment:
Why is OpenCV a requirement?

Reply:
If OpenCV is not installed, one will run into this exception : 

```
Exception in thread "main" java.lang.UnsatisfiedLinkError: /private/var/folders/gf/frgm66d17v76pgmx1t44qjfxxsv1yh/T/mxnet2167090749058732516/mxnet-scala: dlopen(/private/var/folders/gf/frgm66d17v76pgmx1t44qjfxxsv1yh/T/mxnet2167090749058732516/mxnet-scala, 1): Library not loaded: /usr/local/opt/opencv/lib/libopencv_stitching.3.4.dylib
  Referenced from: /private/var/folders/gf/frgm66d17v76pgmx1t44qjfxxsv1yh/T/mxnet2167090749058732516/mxnet-scala
  Reason: image not found
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1824)
	at java.lang.Runtime.load0(Runtime.java:809)
	at java.lang.System.load(System.java:1086)
	at org.apache.mxnet.util.NativeLibraryLoader$.loadLibraryFromStream(NativeLibraryLoader.scala:140)
	at org.apache.mxnet.util.NativeLibraryLoader$.loadLibrary(NativeLibraryLoader.scala:93)
	at org.apache.mxnet.Base$.<init>(Base.scala:70)
	at org.apache.mxnet.Base$.<clinit>(Base.scala)
	at org.apache.mxnet.NDArray$.initNDArrayModule(NDArray.scala:159)
	at org.apache.mxnet.NDArray$.<init>(NDArray.scala:39)
	at org.apache.mxnet.NDArray$.<clinit>(NDArray.scala)
	at org.apache.mxnet.javaapi.NDArray$.ones(NDArray.scala:50)
	at org.apache.mxnet.javaapi.NDArray.ones(NDArray.scala)
	at mxnet.App.main(App.java:10)
```

Reply:
So I believe, it's a dependency for the MXNet package itself.

Reply:
ok, I think we should fix this in the future.